### PR TITLE
Tweaked inserting image to Redactor X content

### PR DESCRIPTION
**Pull request by karelvasicek**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 3.0.1, 2023-10-24
```
Comment (dmitry-mukhin) [inline CHANGELOG.md:9]: I believe this calls for 3.1.0. just in case, since it's not just a patch but refactoring of how things work.

Comment (rsedykh) [reply CHANGELOG.md:9]: I do agree.

**Review by rsedykh — CHANGES_REQUESTED** [review]
Thank you for your contribution! Please bump it to 3.1.0 and I'll merge it. :-)

File: CHANGELOG.md (hunk 1 above)
Comment (karelvasicek) [reply CHANGELOG.md]: > this calls for 3.1.0

Makes sense. Changed.

**Review by rsedykh — APPROVED** [review]

**Pull request merged**